renovate.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
```

Comment:
We should also add `"minimumReleaseAge": "21 days"` to comply with the requirements.

Reply:
@ayodejidev Done